docs/src/api/class-browsercontext.md

```diff
@@ -6,7 +6,7 @@ BrowserContexts provide a way to operate multiple independent browser sessions.
 If a page opens another page, e.g. with a `window.open` call, the popup will belong to the parent page's browser
 context.
 
-Playwright allows creation of "incognito" browser contexts with `browser.newContext()` method. "Incognito" browser
+Playwright allows the creation of "incognito" browser contexts with [`method: Browser.newContext`] method. "Incognito" browser
```

Comment:
how about 'allows the creation of' => 'allows creating' ?

Reply:
changed!